src/docfx/docset/Document.cs

```diff
@@ -271,6 +268,34 @@ public static Document TryCreateFromFile(Docset docset, string path)
             return default;
         }
 
+        // TODO: report the fallback info
+        internal static bool TryResolveDocset(Docset docset, string file, out Docset realDocset)
```

Comment:
Put this to LocalizationConvention as well? otherwise, it is private

Reply:
make it private

---
In reply to: [225767257](https://github.com/dotnet/docfx/pull/3499#discussion_r225767257) [](ancestors = 225767257)